packages/custom-functions-metadata/src/generate.ts

```diff
@@ -151,43 +151,53 @@ type CustomFunctionsSchemaDimensionality = "invalid" | "scalar" | "matrix";
  * @param inputFile - File that contains the custom functions
  * @param outputFileName - Name of the file to create (i.e functions.json)
  */
-export async function generate(inputFile: string, outputFileName: string, wantConsoleOutput: boolean = false): Promise<IGenerateResult> {
-    const errors: string[] = [];
-    const associate: IAssociate[] = [];
-    const generateResults: IGenerateResult = {
-        associate,
-        errors,
-    };
-
-    if (fs.existsSync(inputFile)) {
-        const sourceCode = fs.readFileSync(inputFile, "utf-8");
-        const parseTreeResult: IParseTreeResult = parseTree(sourceCode, inputFile);
-        parseTreeResult.extras.forEach((extra) => extra.errors.forEach((err) => errors.push(err)));
-        generateResults.associate = [...parseTreeResult.associate];
-
-        if (errors.length === 0) {
-            const json = JSON.stringify({ functions: parseTreeResult.functions }, null, 4);
-
-            try {
-                fs.writeFileSync(outputFileName, json);
-
-                if (wantConsoleOutput) {
-                    console.log(`${outputFileName} created for file: ${inputFile}`);
-                }
-            } catch (err) {
-                if (wantConsoleOutput) {
-                    console.error(err);
-                }
-                throw new Error(`Error writing: ${outputFileName} : ${err}`);
-            }
-        } else if (wantConsoleOutput) {
-            console.error("Errors in file: " + inputFile);
-            errors.forEach((err) => console.error(err));
+export async function generate(
+  inputFile: string,
```

Comment:
I personally dislike parameters on different lines.

Reply:
The parameters are on different lines because they exceed the 120 limit.  I don't want to increase the limit and don't know of another change in prettier we could make to allow the parameters to remain on the same line.